documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -147,19 +139,125 @@ Import the following into PGAdmin
             "Port": 15432,
             "MaintenanceDB": "postgres",
             "Username": "postgres",
-            "SSLMode": "prefer",
-            "SSLCompression": 0,
-            "Timeout": 10,
             "UseSSHTunnel": 0,
             "TunnelPort": "22",
-            "TunnelAuthentication": 0
+            "TunnelAuthentication": 0,
+            "KerberosAuthentication": false,
+            "ConnectionParameters": {
+                "sslmode": "prefer",
+                "connect_timeout": 10
+            }
         }
     }
 }
+EOF
 ```
 
-## Deleting GDX Agreements Database
+> To use this file, simply open PGAdmin, click tools > Import / Export Servers > Import,
+> select your pgAdminConfig.JSON file, and import it by following the prompts.
+
+> If you want to configure PGAdmin without a JSON file, simply use the interface to create a new server using the database values from the JSON snippet above, or from **./backend/.env**.
+
+### Run the Backend API
+
+- check which version of Node you should be using.
+
+```bash
+ cd ./backend
+ cat package.json | less
+ # your version should look like this:
+...
+  "engineStrict": true,
+  "engines": {
+    "node": ">=18.18.0"
+  },
+...
+```
+
+- Switch to the correct (currently: v18.18.0) Node version:
+
+```bash
+cd ./backend
+nvm list # find out what versions are available for you
+nvm install  v18.18.0 # install matching version from package.json
+nvm use v18.18.0 # switch to that version
+nvm alias default v18.18.0 # Optional: set the default version
+npm i # installs dependencies
+npm run start # runs the backend API
+```
+
+---
+
+---
+
+## Frontend Setup and Deploy
+
+### Set up Frontend App
+
+- Create`./frontend/.env` using the example below as a template:
+- the `REACT_APP_KEYCLOAK_URL`, `REACT_APP_KEYCLOAK_CLIENT_ID`, `REACT_APP_KEYCLOAK_REALM` will have to be updated according to your [keycloak server settings](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/acd38d-dev/deployments/pmo-app-deployment/yaml)
+
+```bash
+# frontend .env
+WDS_SOCKET_PORT=3000
+REACT_APP_API_URL="https://localhost:8080/"
+REACT_APP_KEYCLOAK_CLIENT_ID=# see note above for value
+REACT_APP_KEYCLOAK_REALM=standard
+REACT_APP_KEYCLOAK_URL=# see note above for value
+```
+
+- check which version of Node you should be using.
+
+```bash
+ cd ./frontend
+ cat package.json | less
+ # your version should look like this:
+...
+  "engineStrict": true,
+  "engines": {
+    "node": ">=18.18.0"
+  },
+...
+```
+
+- Switch to the correct (currently: v18.18.0) Node version:
+
+```bash
+cd ./frontend
+nvm list # find out what versions are available for you
+nvm install  v18.18.0 # install matching version from package.json
+nvm use v18.18.0 # switch to that version
+nvm alias default v18.18.0 # Optional: set the default version
+npm i # installs dependencies
```

Comment:
for the frontend probably should leave the original yarn reference, i noticed that it seems to make a difference.

Reply:
changed `npm run start` to `yarn start`